scripts/Dockerfile.build

```diff
@@ -1,11 +1,12 @@
 FROM golang:alpine
 
-ARG TF_VERSION=1.4.5
+ARG TF_VERSION=1.5.7
```

Comment:
If we are upgrading TF version, I think we should run full regression on existing tests

Reply:
already done, and this is only for local.